doc/lualine.txt

```diff
@@ -209,69 +208,69 @@ CUSTOM OPTIONS FOR COMPONENTS                        *lualine_custom_options*
 Options for components~
 ======================
 
-Global Default options~
+Global options~
 ----------------------
 
-Default options act as default for all components
-
-icons_enabled (Default: true)
-
-  Displays icons on components
-  You should have nerd-fonts supported fonts to see
-  icons properly.
-  Suported by branch, fileformat, filetype, location
-
-  Example:
-    `lualine.options.icons_enabled = true`
-
-Genaral options~
----------------
-These options are available for all components.
+Global options chnge behaviour of all suported components.
```

Comment:
typo: chnge - > change

Reply:
Yeah I just copied from readme, fixed in both